### ISSUE TITLE
In ch 4 it says XFCE is going to qt5, should say gtk3

### ISSUE THREAD

**Issue by kwillett7** [jhajek/Linux-text-book-part-1#10]
Professor Hajek,

I was reading your text book and found a major error in CH 4 when talking about the XFCE 4 environment.  I personally use XFCE and follow it's development somewhat.  They are converting each part from using gtk2 to gtk3.  They are not switching tool kits to qt5 to the best of my knowledge.  Here is a recent news post from an open source news website talking about their progress on the gtk3 port.  https://www.phoronix.com/scan.php?page=news_item&px=Xfce-4.14-GTK3-Port-2
Just thought you might want to know.

**Comment by jhajek** [comment]
I think you might be right.  This was originally written in Aug of 2015, at that time there was rumors of a shift, but this needs to be updated--you are correct.   If you would like to offer an adjustment, please issue a pull request and I will include it.

**Comment by kwillett7** [comment]
I made the changes but couldn't push the changes to the repository.  Returned error 403.

I changed the last line of the XFCE section to read: "Xfce until recently was using the GTK2 toolkit but is in the process of being rewritten in GTK3 conversion started with version 4.12 in 2015.  As of 2018, most of the major components have been re-written in GTK3.  When version 4.14 is released all major components will be converted to GTK3."
